orchestrator/src/test/java/com/yahoo/vespa/orchestrator/resources/ApplicationSuspensionResourceTest.java

```diff
@@ -114,6 +115,7 @@ public void delete_works_on_suspended_and_not_suspended_applications() {
     @Test
     public void list_applications_returns_the_correct_list_of_suspended_applications() {
         // Test that initially we have the empty set
+        var v = webTarget.request(MediaType.APPLICATION_JSON);
```

Comment:
What's the purpose of this? Same for the above `v`.

Reply:
Sorry, debugging hopeless Jersey stuff. Forgot to remove.

Reply:
```suggestion
```